hack/documentation/document.go

```diff
@@ -50,7 +51,11 @@ func WriteDocs() {
 		fmt.Printf("Error encoding: %s\n", err.Error())
 		os.Exit(1)
 	}
-	os.Stdout.Write(b)
+	_, err = os.Stdout.Write(b)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
I'd prefer to continue the use of `logf` (from `sigs.k8s.io/controller-runtime/pkg/log`) since the rest of the framework is using that. I know this particular documentation package isn't running in a pod for logs to be seen, so maybe it's okay to use `log` here. What do you think?

Reply:
the reason i am importing the logx to import log and not logf is to use the log.Fatal. The other package does not have a way to log and exit the program. I would have to use the os.exit which is the same that the log package is using anyways. So either i use os.exit importing os library and coding it myself after using logging the error, or i just import log and do a log.Fatal. comes to about the same thing. i just found it better to use the log.Fatal one. What do you think? @lisa